### PR TITLE
adding a compressible bomex and turbconv_filters 

### PR DESCRIPTION
**Pull request by yairchn**

### CODE REVIEW

File: examples/hybrid/types.jl [hunk 1]
```diff
@@ -69,6 +69,10 @@ function microphysics_model(parsed_args)
     end
 end
 
+function anelastic_dycore(parsed_args)
+    anelastic_dycore = parsed_args["anelastic_dycore"]
+end
+
```
Comment (charleskawczynski) [inline examples/hybrid/types.jl:75]:
```suggestion
```

Comment (yairchn) [reply examples/hybrid/types.jl:75]: sorry this is part of #668 , I am just prototyping here

File: examples/hybrid/callbacks.jl [hunk 2]
```diff
@@ -13,6 +13,15 @@ function get_callbacks(parsed_args, simulation, model_spec, params)
         save_positions = (false, false),
     )
 
+    additional_callbacks = if !isnothing(model_spec.turbconv_model)
```
Comment (charleskawczynski) [inline examples/hybrid/callbacks.jl:16]:
```suggestion
    tc_callbacks = if !isnothing(model_spec.turbconv_model)
```

Comment (charleskawczynski) [reply examples/hybrid/callbacks.jl:16]: And below use

`additional_callbacks = (additional_callbacks..., tc_callbacks...)`

**Pull request closed without merging**